kubernetes/createYaml.sh

```diff
@@ -30,6 +30,14 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
   
   sleep 40s
   
+  kubectl exec -it ignite-alcor-dpm-0 -n ignite-alcor-dpm -c ignite-alcor-dpm-node -- /opt/ignite/apache-ignite/bin/control.sh --activate
+
+  sleep 10s
+
+  kubectl exec -it ignite-alcor-ncm-0 -n ignite-alcor-ncm -c ignite-alcor-ncm-node -- /opt/ignite/apache-ignite/bin/control.sh --activate
+
+  sleep 10s
+
```

Comment:
@pkommoju and @yanmo96 Can we use kubectl command to check the pod status instead of using sleep to wait for the pod created?
@pkommoju Why we only need to active `ignite-alcor-*-0` pod?

Reply:
We are starting only one instance. Ignite requires activate command.

Reply:
Hi @cj-chung  @pkommoju I have create PR #733 to check pod status instead of using sleep. And some other fixes. We can merge my PR after @pkommoju, I will go resolve the conflicts 👍